styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java

```diff
@@ -65,7 +66,8 @@
  */
 class TriggerManager implements Closeable {
 
-  private static final Logger LOG = LoggerFactory.getLogger(TriggerManager.class);
+  private static final Logger DEFAULT_LOG = LoggerFactory.getLogger(TriggerManager.class);
+  private final Logger log;
```

Comment:
we don't need a different logger here. Let's reuse the first one.

Reply:
I added this to be able to pass a mocked logger from the test class to verify that the logging happened

Reply:
Usually verifying logging is not needed. An exception case would be that logging is the interface between systems, like our audit logging

Reply:
Makes sense, I can revert that last commit